custom/ewsghana/reports/specific_reports/stock_status_report.py

```diff
@@ -125,12 +137,24 @@ def calculate_percent(x, y):
 class MonthOfStockProduct(EWSData):
 
     slug = 'mos_product'
-    title = 'Current MOS by Product'
     show_chart = False
     show_table = True
     use_datatables = True
 
     @property
+    def title(self):
+        if not self.location:
+            return ""
+
+        location_type = self.location.location_type.name.lower()
+        if location_type == 'country':
+            return "Current MOS by Product - CMS, RMS, and Teaching Hospitals"
+        elif location_type == 'region':
+            return "Current MOS by Product - RMS and Teaching Hospitals"
+        elif location_type == 'district':
```

Comment:
should this just be `else`? otherwise it will return `None` for an unexpected type